lib/src/priority_queue.dart

```diff
@@ -58,6 +58,13 @@ abstract class PriorityQueue<E> {
   /// an object.
   bool contains(E object);
 
+  /// Provides access to all the elements curently in the queue.
```

Comment:
Mention the 'no particular order' in the leading paragraph as well since that's the essential purpose.

Reply:
That's actually not the essential purpose, the essential purpose is to be efficient.
Being arbitrarily ordered is a means to that goal. If it didn't help us being efficient, we wouldn't care that the order is unspecified.

Reply:
I've update the documentation.